src/backend/distributed/citus.sql

```diff
@@ -42,7 +42,6 @@ CREATE TABLE citus.pg_dist_shard(
     logicalrelid oid NOT NULL,
     shardid int8 NOT NULL,
     shardstorage "char" NOT NULL,
-    shardalias text,
```

Comment:
My understanding is that citus.sql is for new installs of the extension. And regression tests...seem to create a brand new database and extension install on each run.

So, we would want:

1. new installs of citus extension to omit shardalias;
2. regression tests to omit shardalias in DDL (to guard against code referencing the column);
3. allow existing citus installs to keep shardalias column.

Did I get all that right?